webpack.config.js

```diff
@@ -87,7 +90,10 @@ if ( CALYPSO_ENV === 'desktop' || CALYPSO_ENV === 'desktop-mac-app-store' ) {
 	webpackConfig.output.filename = '[name].js';
 } else {
 	webpackConfig.entry.vendor = [ 'react', 'store', 'page', 'wpcom-unpublished', 'jed', 'debug' ];
-	webpackConfig.plugins.push( new webpack.optimize.CommonsChunkPlugin( 'vendor', '[name].[hash].js' ) );
+	if ( CALYPSO_ENV !== 'development' ) {
+		// commons chunk is handy but slows down rebuilds terribly
+		webpackConfig.plugins.push( new webpack.optimize.CommonsChunkPlugin( 'vendor', '[name].[hash].js' ) );
```

Comment:
this shaved a ton of time off the build just on it's own, but for an odd reason. The plugin itself doesn't take much time, but it makes RemoveParentModulesPlugin take nearly twice as much time.